run

```diff
@@ -259,6 +259,7 @@ parse_options(){
         exit 1
     fi
 
+    DISK_OWNER=$OWNER
```

Comment:
Why do we need this variable?

Reply:
This variable is needed to separate NODE and OWNER since below NODE is added to OWNER. This is used on line 689: VOLUMES+=("disk-${disk[0]}-$DISK_OWNER:$disk_path:raw:${disk[1]}:${disk[0]}"). This is necessary so that scisi disks (shared) have the same image for two VMs.